suites/suites/suites.yaml

```diff
@@ -675,11 +675,6 @@ tests:
     tests:
       - cosmo_tester/test_suites/test_blueprints -e puppet -e chef -e docker -e windows
 
-  openstack_blueprints_persistence_recovery:
```

Comment:
do not actually delete these 'test' entries but rather make sure that are not referenced from the`tests_suites` section